packages/manager/src/features/CloudPulse/Utils/utils.ts

```diff
@@ -161,3 +170,71 @@ export const getAllDashboards = (
     isLoading,
   };
 };
+
+/**
+ * @param port
+ * @returns object with isValid boolean and errorMsg string
+ * @description Validates a single port and returns the error message
+ */
+export const isValidPort = (
+  port: string
+): { errorMsg: string | undefined; isValid: boolean } => {
+  let errorMsg = PORTS_ERROR_MESSAGE;
```

Comment:
```suggestion
  let errorMsg = undefined;
```

isn't this assignment is unused?